app/views/curation_concerns/file_sets/upload/_script_templates.html.erb

```diff
@@ -1,8 +1,7 @@
-<!-- TODO: make the upload_set_id available on the presenter -->
-<% if false && presenter.upload_set_id %>
+<% if @presenter && @presenter.upload_set_id %>
```

Comment:
This is the partial that calls it without a presenter: https://github.com/projecthydra/sufia/blob/master/app/views/curation_concerns/file_sets/_multiple_upload.html.erb#L3